modules/node_modules/@ncigdc/containers/FilesCount.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import React from 'react';
+import Relay from 'react-relay';
+
+type TProps = {
+  hits: {
+    pagination: {
+      total: number,
+    },
+  },
+};
+
+const FilesCountComponent = (props: TProps) => (
+  <span>
+    {props.hits.pagination.total > 0
+      ? props.hits.pagination.total.toLocaleString()
+      : <span className="fa fa-spinner fa-spin" />
+    }
+  </span>
+);
+
+export const FilesCountQuery = {
+  fragments: {
+    hits: () => Relay.QL`
+      fragment on FileConnection {
```

Comment:
Looks like `FilesCount` and `CasesCount` are the same thing. I think it would be easier to combine them by changing the query to 

```
fragment ESPagination {
  total
}
```

and then you can reuse it under any endpoint.

But in general I would leave the GraphQL queries